contrib/docker/test/src/DockerModuleTest.scala

```diff
@@ -0,0 +1,69 @@
+package mill
+package contrib.docker
+
+import mill.api.PathRef
+import mill.scalalib.JavaModule
+import mill.util.{TestEvaluator, TestUtil}
+import os.Path
+import utest._
+import utest.framework.TestPath
+
+object DockerModuleTest extends TestSuite {
+
+  object Docker extends TestUtil.BaseModule with JavaModule with DockerModule {
```

Comment:
I've written these based on the other contrib module test suites, let me know if there's anything untoward here.